internal/graphql/resolver/helpers.go

```diff
@@ -13,16 +13,16 @@ import (
 
 var (
 	controlPlaneNamespace = map[model.MeshType]string{
-		model.MeshTypeIstio:       "istio-system",
-		model.MeshTypeLinkerd:     "linkerd-system",
-		model.MeshTypeConsul:      "consul-system",
-		model.MeshTypeOctarine:    "octarine-system",
-		model.MeshTypeTraefikmesh: "traefik-system",
-		model.MeshTypeOsm:         "osm-system",
-		model.MeshTypeKuma:        "kuma-system",
-		model.MeshTypeNginxsm:     "nginx-system",
-		model.MeshTypeNsm:         "nsm-system",
-		model.MeshTypeCitrix:      "ctrix-system",
+		model.MeshTypeIstio:              "istio-system",
```

Comment:
Is this list of namespaces accurate? Perhaps these value should be coming from the individual adapters.

Reply:
Yep, not accurate. Its a scenario that is considered for the time being as we discussed before.